docroot/config/sync/core.extension.yml

```diff
@@ -25,6 +25,9 @@ module:
   history: 0
   image: 0
   link: 0
+  mailchimp: 0
+  mailchimp_campaign: 0
```

Comment:
Do we need mailchimp_campaigns enabled?

Reply:
Yes, because we are going to create campaigns directly in Drupal, in order to send newsletters.